wherehows-web/app/templates/datasets/dataset-compliance/-action-bar.hbs

```diff
@@ -0,0 +1,30 @@
+<section class="action-bar">
+  {{#if isEditing}}
+
+    <div class="container action-bar__content">
+
+      {{#track-ui-event category=trackableCategory.Compliance
```

Comment:
Ps remove the track-ui-event component since this Save or Cancel op is not exclusive to the compliance feature